homeassistant/brands/sonoff

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "sonoff",
+  "name": "Sonoff",
+  "integrations": ["esphome", "tasmota", "zha"]
```

Comment:
I'm not sure if we should add `esphome` and `tasmota` here, as it implies you can just directly use them (which is not the case).

`zha` should not be added as an integration, as it is an IoT standard instead.

Lastly, this PR seems to be untested, as this removes ESPHome, Tasmota & ZHA from the integrations list.

And the PR hasn't been run in a development environment (hassfest didn't apply).